src/m365/base/M365RcJson.ts

```diff
@@ -1,5 +1,6 @@
 export interface M365RcJson {
   apps?: M365RcJsonApp[];
+  context?: any;
```

Comment:
Let's use here the Hash type from src/utils/types.ts instead of any to make it a bit more robust.